<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>94</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>609</width>
    <height>547</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PREFERENCES</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>PREFERENCES_CANCEL</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>PREFERENCES_OK</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="languageLabel">
       <property name="text">
        <string>LANGUAGE</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="languageInput">
       <item>
        <property name="text">
         <string>en_US</string>
        </property>
        <property name="icon">
         <iconset resource="../../assets/resources.qrc">
          <normaloff>:/icons/i18n/en_US.png</normaloff>:/icons/i18n/en_US.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>es_ES</string>
        </property>
        <property name="icon">
         <iconset resource="../../assets/resources.qrc">
          <normaloff>:/icons/i18n/es_ES.png</normaloff>:/icons/i18n/es_ES.png</iconset>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
